Error en la petici√≥n POST: 400
Mensaje de error: {
  "validation_error": {
    "body_params": [
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_indicative": 57,
          "email": "keon37@yopmail.com",
          "name": "Garrett",
          "surname": "Jacobs"
        },
        "loc": [
          "identity_number_company"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      },
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_indicative": 57,
          "email": "keon37@yopmail.com",
          "name": "Garrett",
          "surname": "Jacobs"
        },
        "loc": [
          "country_company"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      },
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_indicative": 57,
          "email": "keon37@yopmail.com",
          "name": "Garrett",
          "surname": "Jacobs"
        },
        "loc": [
          "password"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      },
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_indicative": 57,
          "email": "keon37@yopmail.com",
          "name": "Garrett",
          "surname": "Jacobs"
        },
        "loc": [
          "post"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      }
    ]
  }
}
